outline.md

```diff
@@ -0,0 +1,90 @@
+# An Oriole for Inner Source
+45 min
+
+## Abstract
+
+_Note: This abstract was created in February 2018 for submission of this talk to [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/) and is already posted there_.
+
+Inner source is the application of open source methodologies to internally-developed software.
+While simple to define, inner source can be difficult to explain and implement successfully.
+This presentation introduces the key terms, concepts, and principles for effective inner sourcing along with explanations and real examples.
+It is content that is accessible to newcomers while at the same time refreshing to those with experience.
+The same information in this presentation is also delivered as an O’Reilly Oriole, available freely at https://www.safaribooksonline.com/oriole/.
+We’ll review the Oriole, how to use it, and other Orioles related to inner source that are on the way.
+
+## Background
+Many individuals that are excited about inner source share their experience in [the InnerSource Commons](http://innersourcecommons.org/) and other public locations.
+In my experience it's been reasonably difficult for inner source practitioners to share ideas and guidance that are directly usable by others.
```

Comment:
This seems like a key problem you are seeking to fix with these presentations. You want to deliver directly actionable ideas and guidance. It's helpful for me to read this document with this idea in mind.